Apps/frontend/cypress/e2e/3-slider/amplitude.cy.js

```diff
@@ -0,0 +1,27 @@
+describe("amplitudeSlider", () => {
+  beforeEach(() => {
+    cy.visit("http://localhost:5173/");
+  });
+
+  it("is visible", () => {
```

Comment:
Is it possible to get the ten amplitudeSliders in a more modular way (i.e. for loop for all i = 0 < NUM_CHANNELS instead of writing down all ten individually?
Maybe we can use something like this: https://stackoverflow.com/a/53487016

Reply:
yeah good point. changed it.